src/components/List.js

```diff
@@ -107,33 +155,37 @@ const List = ({ token }) => {
       {errorMessage && <Error errorMessage={errorMessage} />}
 
       <ul style={{ listStyleType: 'none' }}>
-        {docs &&
-          docs
-            ?.filter((doc) =>
-              doc?.itemName
+        {sortedList &&
+          sortedList
+            ?.filter((item) =>
```

Comment:
I think renaming `doc` to item is a bit more clear since here we are dealing with the individual list item(s).